city_metrix/layers/building_classifier/V2-building-class-data.geojson

```diff
@@ -0,0 +1,303 @@
+{
```

Comment:
It is only 184 KB so not very large. I like having it here since it is the basis for the model. If we start needing to use more training data in the future, we should find a better solution probably.